controller/api/destination/endpoint_translator.go

```diff
@@ -474,6 +528,18 @@ func (et *endpointTranslator) sendClientRemove(set watcher.AddressSet) {
 	}
 }
 
+func (et *endpointTranslator) altFamily(setID watcher.ID, fam corev1.IPFamily) (*watcher.ID, *watcher.Address) {
+	for id, addr := range et.availableEndpoints.Addresses {
```

Comment:
`diffAddresses` opperates on `et.filteredSnapshot` while this checks `availableEndpoints`.  Could this lead to incorrect diffs when zone filtering is on?

Reply:
Good point. I'll refactor this to consume the filtered addresses instead.